protos/sigstore_common.proto

```diff
@@ -168,25 +168,17 @@ message SubjectAlternativeName {
         }
 }
 
-// A chain of X.509 certificates.
+// A collection of X.509 certificates.
+//
+// NOTE: "Chain" is a misnomer in this context, since there is no one true certificate chain
+// in most PKI schemes. This message should be treated as a generic collection of certificates
+// for path construction.
```

Comment:
Flagging: I added this `NOTE` based on the discussion we had in the clients call -- it might be overkill, though.

Reply:
It's not just used for path construction though, also for providing a root CA certificate for TUF use cases.

Maybe add proposed different use cases?